### PR TITLE
Implement other rest functions fix minor issues

### PR DESCRIPTION
**Pull request by hjmodi**

### CODE REVIEW

**Review by ligangty — COMMENTED** [review]
Generally looks good with a little stuff to change.
For the test, is it just unit testing or some part of the ftests in indy side migration?

File: src/main/java/org/commonjava/indy/service/tracking/client/content/ContentService.java [hunk 1]
```diff
@@ -34,4 +36,8 @@
     @Path( "/tracking/recalculate" )
     Response recalculateEntrySet( final Set<ContentTransferDTO> entries );
 
+    @POST
+    @Path( "/repo/zip" )
+    File getZipRepository( final TrackedContent record );
```
Comment (ligangty) [inline src/main/java/org/commonjava/indy/service/tracking/client/content/ContentService.java:41]: As there are some changes in indy content admin service for this API param, should we change it here accordingly?

Comment (hjmodi) [reply src/main/java/org/commonjava/indy/service/tracking/client/content/ContentService.java]: oh yes! updated it to use slimmer DTOs as in indy monolith

**Comment by hjmodi** [comment]
> Generally looks good with a little stuff to change. For the test, is it just unit testing or some part of the ftests in indy side migration?

I am planning to do a separate PR for unit tests and also for functional tests

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**